test_suite/cloud/test_aws.py

```diff
@@ -321,7 +321,7 @@ def test_max_cstate_is_configured_in_cmdline(self, host):
                     assert not host.file('/proc/cmdline').contains(line), \
                         f'{line} must not be specified in AMIs that are not SAP'
 
-    @pytest.mark.run_on(['rhel', 'centos', 'fedora36'])
+    @pytest.mark.run_on(['rhel7.9', 'rhel8.4', 'rhel8.5', 'rhel9.0', 'centos', 'fedora36'])
```

Comment:
Shouldn't we remove ` 'rhel8.4'` from the list?

Reply:
I'm going to blame this error on the heat... Thanks for noticing!